.github/workflows/build-bottles.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         os: [ { id: macos-10.15, name: "catalina" } ]
         # we don't bottle meta-formulas that contain only services
-        formula: [tezos-accuser-013-PtJakart, tezos-admin-client, tezos-baker-013-PtJakart, tezos-client, tezos-codec, tezos-node, tezos-sandbox, tezos-signer]
+        formula: [tezos-accuser-013-PtJakart, tezos-admin-client, tezos-baker-013-PtJakart, tezos-client, tezos-codec, tezos-node, tezos-signer]
```

Comment:
014 daemons should be added here as well. Annoyingly, there is no easy way to avoid this copy-paste:(

Reply:
change squashed into initial update brew commit